projects/canopy/src/lib/shadow/shadow.directive.ts

```diff
@@ -1,9 +1,14 @@
-import { Directive, HostBinding } from '@angular/core';
+import { Directive, HostBinding, input } from '@angular/core';
 
 @Directive({
   selector: '[lgShadow]',
   standalone: true,
 })
 export class LgShadowDirective {
+  hasHoverState = input(false);
+
   @HostBinding('class.lg-shadow') class = true;
+  @HostBinding('class.lg-shadow-hover') get hoverClass() {
```

Comment:
We use BEM in Canopy so I'd write this class as `lg-shadow--hover`

Reply:
Thanks for the comments. I had the same dilemma. I am leaning towards verb, _when you hover_. I'll wait for confirmation from @elenagarrone.

Reply:
In this case is a modifier to the regular look of the `lg-shadow` 🤔